Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -227,22 +547,29 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
             });
             Y.Mock.expect(this.version, {
                 method: 'set',
-                args: ['fields', this.fields]
+                args: ['fields', Y.Mock.Value.Object]
```

Comment:
Simple object for fields is being passed in tests, but in app it seems that it contains whole `__proto__` (which simple object in tests doesn't have), so objects are not exactly the same. I have added an assertion in `run` function to check if important attributes are same.